src/components/ListItem/ListItem.vue

```diff
@@ -28,6 +28,9 @@
                   :data-test-id="`${dataTestId}-toggle`"
                   :disabled="disabled"
     />
+    <strong v-if="!hasToggle && itemValue"
```

Comment:
(nit) please consolidate these 3 lines onto a single line, like so: 

```js
const computed = {
   ...
  isRightLabelShowing() {
    const result = (!this.hasToggle && this.rightLabel);
    return result;
  },
};

...

<strong v-if="isRightLabelShowing" class="right-label">{{ rightLabel }}</strong>
```
And please refactor the `v-if` evaluation into a new well-named `computed` prop called: `isRightLabelShowing`